FrameMetadata/FrameMetadata/ViewController.m

```diff
@@ -0,0 +1,319 @@
+//
+//  ViewController.m
+//  Lets-Build-Publisher
+//
+//  Copyright (c) 2013 TokBox, Inc. All rights reserved.
+//
+
+#import "ViewController.h"
+#import <OpenTok/OpenTok.h>
+#import "TBExampleVideoCapture.h"
+#import "TBExampleVideoRender.h"
+
+@interface ViewController ()
+<OTSessionDelegate, OTSubscriberKitDelegate, OTPublisherDelegate, TBRendererDelegate>
+
+@end
+
+@implementation ViewController {
+    OTSession* _session;
+    OTPublisher* _publisher;
+    OTSubscriber* _subscriber;
+    
+    TBExampleVideoCapture* _defaultVideoCapture;
+    
+    TBExampleVideoRender* _subscriberVideoRenderView;
+    TBExampleVideoRender* _publisherVideoRenderView;
+    
+    UILabel *metadataLabel;
+}
+static double widgetHeight = 240;
+static double widgetWidth = 320;
+
+// *** Fill the following variables using your own Project info  ***
+// ***          https://dashboard.tokbox.com/projects            ***
+// Replace with your OpenTok API key
+static NSString* const kApiKey = @"";
+// Replace with your generated session ID
+static NSString* const kSessionId = @"";
+// Replace with your generated token
+static NSString* const kToken = @"";
+
+#pragma mark - View lifecycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    // Step 1: As the view comes into the foreground, initialize a new instance
+    // of OTSession and begin the connection process.
+    _session = [[OTSession alloc] initWithApiKey:kApiKey
+                                       sessionId:kSessionId
+                                           delegate:self];
+    
+    metadataLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 100, 30)];
+    [metadataLabel setTextColor:[UIColor whiteColor]];
+    [metadataLabel setFont:[UIFont systemFontOfSize:12]];
+    
+    [self doConnect];
+}
+
+- (BOOL)prefersStatusBarHidden
+{
+    return YES;
+}
+
+- (BOOL)shouldAutorotateToInterfaceOrientation:
+(UIInterfaceOrientation)interfaceOrientation
+{
+    // Return YES for supported orientations
+    if (UIUserInterfaceIdiomPhone == [[UIDevice currentDevice]
```

Comment:
You could do
`return UIUserInterfaceIdiomPhone != [[UIDevice currentDevice] userInterfaceIdiom]`